frontend/views/components/AvatarUser.vue

```diff
@@ -41,8 +41,8 @@ export default {
         console.error(`AvatarUser: ${this.username} doesn't exist!`)
         return
       }
-      const state = await sbp('state/latestContractState', userContractId)
-      this.ephemeral.url = state.attributes.picture
+      const state = await sbp('state/latestContractState', userContractId) || {}
+      this.ephemeral.url = state.attributes && state.attributes.picture
```

Comment:
Why this change?

Reply:
because I just crossed with a weird bug where this sbp() got called and it didn't return a proper state. To prevent JS from breaking, I added the validation.

Reply:
OK. I asked because sometimes it helps to let things break to track down bugs. By doing stuff like this, it's possible for bugs to be hidden (i.e. something is a way it shouldn't be). I guess this isn't a big deal, but in the future please try to track down the bug, or at least document it in a comment/issue if you can.

Reply:
Will do. In this case, I looked into it to try to replicated and it happened because I erased the local Application data while logged in. Now it can't retrieve the user attributes anymore. Once I create a new account it didn't happen anymore. I assume this won't happen in production :)